specs/support/view_examples.rb

```diff
@@ -12,18 +10,22 @@ class NewProductUser < Spectacles::View
 
   describe "Spectacles::View" do
     describe "inherited class" do
-      it "can has scopes" do
+      before(:each) do
         User.destroy_all
         Product.destroy_all
         @john = User.create(:first_name => 'John', :last_name => 'Doe')
         @john.products.create(:name => 'Rubber Duck', :value => 10)
+      end
+
+      let(:new_product_user) { NewProductUser.duck_lovers.load.first }
 
-        NewProductUser.duck_lovers.load.first.username.must_be @john.first_name
+      it "can have scopes" do
+        _(new_product_user.username).must_be @john.first_name
       end
 
       describe "an instance" do
         it "is readonly" do
-          NewProductUser.new.readonly?.must_be true
+          _(new_product_user.readonly?).must_be true
```

Comment:
This avoids an internal deprecation warning that resulted from calling `.new` on `NewProductUser`.

Reply:
👍